File: GitHub/Java/Sphere.java
import java.util.*;

public class Sphere {
	public static void main(String[] args){
		Scanner start = new Scanner(System.in);
		System.out.println("Enter the radius of your sphere:");
		double radius = start.nextDouble();
		double volume = (4.0/3.0) * Math.pow(radius, 3) * Math.PI;
		double surfarea = 4 * Math.pow(radius, 2) * Math.PI;
		System.out.println("The volume of your sphere is:");
		System.out.println(volume);
		System.out.println("The surface area of your sphere is:");
		System.out.println(surfarea);
	}
}

File: GitHub/Python/Intro/sphere.py
import math

def main():
    radius = eval(input("Enter the radius of the sphere: "))
    volume = 4.0/3.0 * math.pi * radius ** 3
    area = 4 * math.pi * radius ** 2
    print("The surface are of your sphere is:", area)
    print("The volume of your sphere is:", volume)

main()
